### ISSUE TITLE
consider adding a "us" geo_type to covidcast

### ISSUE THREAD

**Issue by sgratzl** [cmu-delphi/delphi-epidata#181]
atm the highest level is `state` but afaik there is no option to get signal values for the us as a whole. It would be nice to have them since it would provide a nice starting point for a visual drill down operation

**Comment by krivard** [comment]
Agree; FB asked for this back in July but we weren't in a position to provide it then.

Propose instead the following:
* `geo_type`: `nation`
* `geo_value`: `usa`

This will need to be handled in each indicator unless we want to get clever with aggregating aggregates. Moving to the indicators repo for further work.
